### ISSUE TITLE
fw

### ISSUE THREAD

**Issue by Dikosh** [Rahulvks/Text-Classification-and-Prediction-Using-XGBoost#1]
Hello, can You help me? I tried to run your code, but in Xgboost models I get an error.
`xgbmodel <- xgboost(data = ctrain, max.depth = 25, eta = 0.3, nround = 200, objective = "multi:softmax", num_class = 20, verbose = 1, watchlist = watchlist)
Error in check.custom.obj() : 
  Setting objectives in 'params' and 'obj' at the same time is not allowed
`

**Comment by Rahulvks** [comment]
Hi Dikosh
Can you elaborate little bit more ?

**Comment by Dikosh** [comment]
I just copied whole your code and try to run, but when I try run xgboost  models I get an error. Please can you run it in yourself computer? I think that there is small updates in Xgboost packages. I try to fix and understand it. But all my attemts was unsuccessful.

**Comment by Rahulvks** [comment]
Okay i Will update you tomorrow. Did you tried same test and train from github ?

**Comment by Dikosh** [comment]
Thank you very much, I will wait you. Yes same test and train data from github

**Comment by Dikosh** [comment]
Hello! Can you see it please?

**Comment by Rahulvks** [comment]
Hi DIkosh

It's working fine for me. please update R studio and library and try once again.

**Comment by Dikosh** [comment]
Hi!
I just updated packages for older one, and it works! Thanks! 
One more question, how I can get f-measure score from this model? not accuracy, thanks!
